CHANGELOG.md

```diff
@@ -25,6 +30,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Added
 
 - **CUMULUS-2185** - RDS Migration Epic
+  - **CUMULUS-2455**
+    - Move granules API endpoint records move updates for migrated granule files
```

Comment:
Need to note the additions/updates to the ingest granules package API

Reply:
Nit: Can we attempt to put this in the CHANGELOG in ascending ticket order? I'm sure that we're not being consistent, but still

Reply:
I'm also not sure what this line is trying to say. Are we trying to communicate that the move granules API now captures updates into PostgreSQL as well?

Do we need to flag anywhere the overall changes to the logic in the move granule API?

Reply:
re: ascending ticket order - we sure can. 

`I'm also not sure what this line is trying to say. Are we trying to communicate that the move granules API now captures updates into PostgreSQL as well?`

Fair, that is a bit unclear.    Perhaps this helps? 5e3882d99  

`Do we need to flag anywhere the overall changes to the logic in the move granule API?`

It's probably worth taking a look at the API docs to see if the underlying behavior is covered in any detail.

Reply:
Update for sort/deprecation notice: 1e8157a67